applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_apply_scalar_constraint_table_process.cpp

```diff
@@ -0,0 +1,127 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_processes/apply_scalar_constraint_table_process.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+
+#include "geo_mechanics_application_variables.h"
+
+namespace
+{
+
+using namespace Kratos;
+using namespace Kratos::Testing;
+
+void AssertNodesHaveCorrectValueAndFixity(double expected_value, bool expected_fixity, const ModelPart& rModelPart)
+{
+    for (const auto& r_node : rModelPart.Nodes()) {
+        KRATOS_EXPECT_EQ(r_node.IsFixed(DISPLACEMENT_X), expected_fixity);
+        KRATOS_EXPECT_EQ(r_node.FastGetSolutionStepValue(DISPLACEMENT_X), expected_value);
+    }
+}
```

Comment:
The parameter names need to be changed to follow the Kratos Style Guide. Also, we only need access to the node container (not the complete model part), so we may want to change the final parameter type to reflect that.
```suggestion
void AssertNodesHaveCorrectValueAndFixity(double expected_value, bool expected_fixity, const ModelPart::NodesContainerType& rNodes)
{
    for (const auto& r_node : rNodes) {
        KRATOS_EXPECT_EQ(r_node.IsFixed(DISPLACEMENT_X), expected_fixity);
        KRATOS_EXPECT_EQ(r_node.FastGetSolutionStepValue(DISPLACEMENT_X), expected_value);
    }
}
```

At the calling end, we need to pass `r_model_part.Nodes()`.

Reply:
Good point, done!